_articles/features/authenticator-keys.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: article
+title: Authenticator key (TOTP) storage and use
+categories: [features]
+featured: true
+popular: false
+tags: [autofill, auto-fill, totp, 2fa, two-step login, two factor authentication, authenticator]
+---
+
+{% note %}Authenticator key (TOTP) storage is available to all accounts. TOTP code generation requires a premium membership or paid organization account.{% endnote %}
+
+Each website that supports TOTP or 2FA with an "Authenticator" handles configuration a little differently. You will need to start the setup from each individual website or service that you are accessing (e.g. google.com, github.com). The option to configure this will commonly be found under the "Security" options of your account.
```

Comment:
You may want to link TOTP and 2FA to wikipedia articles. It might also make sense to write out their full names the first time you mention them and then put the abbreviations in parens.